junit-tests/src/test/java/org/junit/gen5/engine/DummyTestEngineDescriptor.java

```diff
@@ -10,20 +10,21 @@
 
 package org.junit.gen5.engine;
 
-class DummyTestDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
-
-	public static final String ENGINE_ID = "dummy";
-
+class DummyTestEngineDescriptor extends AbstractTestDescriptor implements Leaf<DummyEngineExecutionContext> {
```

Comment:
Well, at the moment it is used for the engine node and its children. The unique IDs are build like this: `ENGINE_ID + ":someString"`. I agree that this could be cleaned up, but would prefer the original class name.